tests/fixtures/proxy-i18n-skip-normalize/proxy.ts

```diff
@@ -0,0 +1,22 @@
+import type { NextRequest } from 'next/server'
+import { NextResponse } from 'next/server'
+
+export async function proxy(request: NextRequest) {
+  const response = NextResponse.next()
+
+  if (response) {
+    response.headers.append('Deno' in globalThis ? 'x-deno' : 'x-node', Date.now().toString())
+    // report Next.js Middleware Runtime (not the execution runtime, but target runtime)
+    // @ts-expect-error EdgeRuntime global not declared
+    response.headers.append('x-runtime', typeof EdgeRuntime !== 'undefined' ? EdgeRuntime : 'node')
+    response.headers.set('x-hello-from-middleware-res', 'hello')
+
+    response.headers.set('x-next-url-pathname', request.nextUrl.pathname)
+
+    return response
+  }
+}
+
+export const config = {
+  runtime: 'nodejs',
+}
```

Comment:
It seems like Proxy will be Node.js only (tho it's not as of now), so I do add this explicitly here in hope the test setup won't break when Next.js ships the change to make Proxy Node.js only. If that change end up not shipping, I'll adjust the setup to run in both Edge and Node.js runtimes as we do for most of our middleware fixtures